Source/Commands.c

```diff
@@ -343,15 +343,11 @@ static void sayCommand(Server* server, char command[30], char* message, uint8 pl
 
 static void banIPCommand(Server* server, char command[30], char* message, uint8 player)
 {
-    char         ipString[16];
-    int          ip[4]; // Waste of memory. FIXME later
-    unsigned int ip32;
+    char    ipString[16];
+    IPUnion ip;
```

Comment:
Wanted to do this ages ago when i added the Union but forgot about it completely haha. Thanks for that.